scripts/ticketmaster-API.py

```diff
@@ -17,63 +17,96 @@
-                  main_venue_name, main_venue_lat, main_venue_lon,
-                  main_artist_id, main_artist_name, main_artist_genre)
-
-        cursor.execute(query, values)
-        db.commit()
-
-    time.sleep(1)
+    pages_num = data['page']['totalPages']
+    if pages_num == 1:
+        pages_list = [0]
+    else:
+        pages_list = range(0,pages_num-1)
+
+    #print(us_state)
+    #print(pages_num)
+
+    entry = 1
+
+    for page in pages_list:
+        params = {'stateCode': us_state,
+                  'classificationName': classificationName,
+                  'apikey': apikey,
+                  'startEndDateTime': ['2019-05-01T00:00:00Z',
+                                       '2019-05-31T23:59:59Z'],
+                  'size': size,
+                  'page': page
+                  }
+
+        r = requests.get(url=URL, params=params)
+
+        data = r.json()
+
+        events = data['_embedded']['events']
+
+        #print(us_state,page)
+
+        for index, event in enumerate(events):
+            local_date = event['dates']['start']['localDate']
+            event_id = event['id']
+
+            main_genre = event['classifications'][0].get('genre')
+            main_genre_name = main_genre['name'] if main_genre else ''
+            sub_genre = event['classifications'][0].get('subGenre')
+            sub_genre_name = sub_genre['name'] if sub_genre else ''
+
+            main_venue = event['_embedded']['venues'][0]
+            main_venue_name = main_venue['name']
+
+            main_venue_location = main_venue.get('location')
+            main_venue_lat = main_venue['location'][
+                'latitude'] if main_venue_location else None
+            main_venue_lon = main_venue['location'][
+                'longitude'] if main_venue_location else None
+
+            artists = event['_embedded'].get('attractions')
+            main_artist_id = artists[0]['id'] if artists else ''
+            try:main_artist_name = artists[0]['name'] if artists else ''
+            except: main_artist_name = ''
+            try: main_artist_genre = artists[0]['classifications'][0]['genre']['name'] if artists else ''
+            except: main_artist_genre = ''
+
+            query = """INSERT INTO ticketmaster_events
+                        (ticketmaster_id, local_date, event_genre,
+                        event_subgenre, venue, venue_lat, venue_long,
+                        artist_id, artist_name, artist_genre) VALUES
+                        (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)"""
+            values = (event_id, local_date, main_genre_name, sub_genre_name,
+                      main_venue_name, main_venue_lat, main_venue_lon,
+                      main_artist_id, main_artist_name, main_artist_genre)
+
+            cursor.execute(query, values)
+            db.commit()
+            print("page",page,"   ","entry",entry,"   ","artist",main_artist_name)
```

Comment:
Same re: print statement and debugging lines. 
You also don't need the variable `entry` here, you have access to the index because this for loop has `enumerate()`

Reply:
Removed debugging.